nominees/opencollective.json

```diff
@@ -0,0 +1,31 @@
+{
+  "name": "Open Collective",
+  "description": "Open Collective is an online funding platform for open and transparent communities. We provide the tools to raise and share your finances in full transparency.",
+  "website": "https://opencollective.com/",
+  "license": [
+    {
+      "spdx": "MIT",
+      "licenseURL": "https://github.com/opencollective/documentation/blob/v2/LICENSE"
+    }
+  ],
+  "SDGs": [
+    {
+      "SDGNumber": 17,
+      "evidenceURL": "https://opencollective.com/community"
```

Comment:
In as much as possible for the `evidenceURL` field try to point to external blog or news articles that validate the SDG in question, take for example this article on [Medium](https://medium.com/open-collective/what-is-open-collective-how-it-works-c51bacc73d94). @ijeomaemeruwa could you update this field to use the Medium article instead or any other stronger one you may find. Thanks.